cartography/intel/aws/ec2/network_interfaces.py

```diff
@@ -0,0 +1,182 @@
+import logging
+import re
+
+from .util import get_botocore_config
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_network_interface_data(boto3_session, region):
+    client = boto3_session.client('ec2', region_name=region, config=get_botocore_config())
+    paginator = client.get_paginator('describe_network_interfaces')
+    subnets = []
+    for page in paginator.paginate():
+        subnets.extend(page['NetworkInterfaces'])
+    return subnets
+
+
+def load_network_interfaces(neo4j_session, data, region, aws_account_id, aws_update_tag):
+
+    elb_associations = []
+    instance_associations = []
+
+    for n_idx, network_interface in enumerate(data):
```

Comment:
Nit: if `n_idx` is never used, do we need `enumerate()`?

Reply:
yes, finally this is not required, there was a version where I modified data in the loop, I'll remove